games-action/minetest/minetest-0.4.12.ebuild

```diff
@@ -0,0 +1,136 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+inherit cmake-utils eutils gnome2-utils user vcs-snapshot
```

Comment:
This is ___very___ bad. We are breaking tree consistency here, so users cannot expect games ebuilds to behave the way they used to behave anymore. That's deliberately breaking tree consistency, I'd say and as such a serious QA issue. No offence to you @damiel

I don't like games.eclass much either, but that's why I work on non-games.eclass ebuilds __outside__ of the tree.